go/src/koding/remoteapi/models/j_group_role.go

```diff
@@ -16,7 +16,7 @@ type JGroupRole struct {
 	// id
 	ID string `json:"_id,omitempty"`
 
-	// is configurable
+	// is configureable
```

Comment:
I believe we are not using that information anymore, it can be removed from `JRole` schema.